ppr-ui/src/components/parties/Parties.vue

```diff
@@ -178,6 +181,13 @@ export default defineComponent({
       })
     })
 
+    onMounted(async () => {
+      if(isSecurityActNotice.value) {
+        const party = await partyCodeAccount(true)
```

Comment:
Does the parent have loading indicator for this, would be nice to have if the call is slow.

Reply:
The parties components (tables) have a template that renders when there is no data, as with all of the table components. Once that data is available, it populates. It's milliseconds.
 
This would be a design question and something to look at holistically, probably not specifically for one instance of a table when data is loading, but all of them and how we handle it. 

I would recommend creating an enhancement ticket and we can discuss as a team, with design!

Reply:
Thanks for explaining, doesn't sound like an issue, i'm good.

Reply:
Copy that, it is a valid topic though. Sometimes the swapping of templates in the dom can be jarring. 
Not all the vuetify components transition perfectly out of the box and sometimes we don't always leverage whats there.

Always worth a talk with design to come up a general approach!